spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -771,9 +774,9 @@ def setup_editor(self,
                      tab_mode=True,
                      strip_mode=False,
                      intelligent_backspace=True,
-                     automatic_completions=True,
-                     automatic_completions_after_chars=3,
-                     automatic_completions_after_ms=300,
+                     automatic_completions=None,
+                     automatic_completions_after_chars=None,
+                     automatic_completions_after_ms=None,

```

Comment:
These changes should be reverted. They are only used when testing CodeEditor instances independently from the editor, but they can be changed if needed by modifying CodeEditor's attributes associated with them directly.